modular_skyrat/modules/interaction_menu/code/interaction_component.dm

```diff
@@ -113,4 +130,89 @@
 			interact_next = interaction_component.interact_last + INTERACTION_COOLDOWN
 			interaction_component.interact_next = interact_next
 			return TRUE
+	if(params["slot"])
+		// This code should be easy enough to follow... I hope.
+		var/item_index = params["slot"]
```

Comment:
Not sure this is the most intuitive name, maybe `item_slot` or something? It makes me think it's going to be an item itself, but it's not.